lang/go/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Remove some binary applications
```

Comment:
I noticed that prometheus package created a binary on go build, that can be added here as well.